src/handy.jl

```diff
@@ -296,12 +298,12 @@ function mysql_bind_array(typs, params)
     length(typs) != length(params) && throw(MySQLInterfaceError("Length of `typs` and `params` must be same."))
     bindarr = MYSQL_BIND[]
     for (typ, val) in zip(typs, params)
-        #Is the value one of three different versions of Null?
-        if (isdefined(:DataArrays)&&(typeof(val)==DataArrays.NAtype))||(isdefined(:NullableArrays)&&(typeof(val)<:Nullable)&&(val.isnull))||(val==nothing) 
+        #Is the value one of four different versions of Null?
+        if (isdefined(:Missings)&&(typeof(val)==Missings.Missing))||(isdefined(:DataArrays)&&(typeof(val)==DataArrays.NAtype))||(isdefined(:NullableArrays)&&(typeof(val)<:Nullable)&&(val.isnull))||(val==nothing)
```

Comment:
No need to support `NA`, DataArrays has been ported to `Missing`. `Nullable` has also been removed from Base, so probably not worth supporting either.